addons/mail/static/src/messaging/component/chat_window/chat_window.js

```diff
@@ -26,6 +26,10 @@ class ChatWindow extends Component {
                 localeTextDirection: this.env.messaging.locale.textDirection,
             };
         });
+        /**
+         * Useful to prevent click on header from wrongly focusing the window.
```

Comment:
Would add sentence of what, e.g. "Reference of header of chat window.".
Seems obvious from looking at statement, but this could become more abstract in the long-term, and it's good for documentation to super-briefly tell the "what" then the "why". Having a documentation with just the "why" looks aggressive, likely to intimidate a newbie developer and, in the end, demoralize.

Reply:
ok